### DISCUSSION TITLE
Cant export config

### DISCUSSION THREAD

**Discussion by madmaxpowermx** [matteobruni/tsparticles#5089] [Q&A]
Hello there, Im trying to use tsParticles on wordpress, using the premiun addons pro plugin. Following their tutorial, but when I click on MORE / EXPORT CONFIG nothing happens. 

In their tutorial appears a popup with the code but it seems is not working. 

How can I get this code?

ty!

**Comment by matteobruni** [comment]
This is an issue of the website, please open an issue there: https://github.com/tsparticles/website

**Comment by madmaxpowermx** [comment]
ok ty
